bleak/backends/bluezdbus/scanner.py

```diff
@@ -304,4 +303,34 @@ def parse_msg(self, message):
         )
 
         if self._callback is not None:
-            self._callback(message)
+            discovery_data = message.body[1]
+
+            # Get all the information wanted to pack in the advertisement data
+            _address = discovery_data["Address"]
+
+            _local_name = discovery_data.get("Name", "Unknown")
```

Comment:
Also should be using `"Alias"` instead of `"Name"`

Reply:
There is also a new experimental "AdvertisementData" property in BlueZ that we could use to get the local name instead of assuming the "Name" property came from the advertising data (which it may or may not have).

https://github.com/bluez/bluez/blob/master/doc/device-api.txt#L260

We don't have to implement this now, but we could add a TODO comment to remind us about it.